react.js

```diff
@@ -1,23 +1,49 @@
 // React Assessments
 
-// 1. Write a simple React component that simply prints "I am a component" to the screen. Be sure to include all necessary imports, exports, etc...
 
-// 2. Practice With Loops: In React, we often use the map function to iterate over an array held in state. To get more practice with this, write a vanilla javascript for loop that outputs the same thing as the map function below. 
+// 1. Write a simple React component that simply prints "I am a component" to the screen. Be sure to include all necessary imports, exports, etc...
+import React, { Component } from './App.js'
+
+export default class Print extends Component {
+  constructor(){
+    super()
+      this.class = {
+        string = 'I am a component'
+      }
+    render(){
+      return()
+        <div>
+          {this.class}
+        </div>
+      }
+    }
+  }
+        //After, in the App.js file, I would import Print from react.js (this file) and call <Print /> in the return
+
+// 2. Practice With Loops: In React, we often use the map function to iterate over an array held in state. To get more practice with this, write a vanilla javascript for loop that outputs the same thing as the map function below.
 
 
 var stuffArray = [42, "Arthur Dent", 1978, "Zaphod", "Vogon", "Marvin, the Paranoid Android"]
 
 stuffArray.map(function(el, i){
   console.log(el + " is at index: " + i)
 })
+  let newArr = () => {
+  for (let i=0; i < stuffArray.length; i++){
+  return newArr.push()
+  console.log(newArr([i]) + 'is at index' + i);
+  }
+}
 
-// your for loop here, you can use the same stuffArray array
 
+// 3. Basic sorting: Find the cheapest price -- you are given a list of prices and need to find the single lowest price from the array. Write a function that takes in an array of numbers, and returns the lowest price
 
-// 3. Basic sorting: Find the cheapest price -- you are given a list of prices and need to find the signle lowest price from the array. Write a function that takes in an array of numbers, and returns the lowest price
+const numList = [13.99, 12.45, 98.34, 20.85, 2.85, 9.67]
 
-var steal = function(){
+function getLowNumber(){
```

Comment:
I think you're very close here. It might be the order in which you're doing things.